vms/platformvm/block/executor/proposal_block_test.go

```diff
@@ -541,15 +530,11 @@ func TestBanffProposalBlockUpdateStakers(t *testing.T) {
 			},
 		},
 		{
-			description:   "advance time to staker5 end",
+			description:   "advance time to staker5 start",
 			stakers:       []staker{staker1, staker2, staker3, staker4, staker5},
 			advanceTimeTo: []time.Time{staker1.startTime, staker2.startTime, staker3.startTime, staker5.startTime},
 			expectedStakers: map[ids.NodeID]stakerStatus{
 				staker1.nodeID: current,
-
-				// given its txID, staker2 will be
-				// rewarded and moved out of current stakers set
-				// staker2.nodeID: current,
```

Comment:
the comment is not really true. We don't generate a proposal block with RewardValidatorTx to remove staker2